doc/source/index.rst

```diff
@@ -15,3 +15,9 @@ free and open-source solar data analysis environment for Python.
  
 .. warning::
     Having issues with links on this page? Make sure there is a trailing slash on the URL *e.g.* `http://docs.sunpy.org/en/latest/`
+    
+
+Link to SunPy
+-----------------
+    
+Wish to stay update with the latest releases? Here is the Link *e.g.* `http://sunpy.org`
```

Comment:
I would prefer something more accessible from everywhere, like in [astropy](http://astropy.readthedocs.org/), where it's shown in a top bar.